lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
```

Comment:
Thanks for the attention you gave to this PR. If you could start here (and maybe fix other places later), that'll be great.